### PR TITLE
HWKALERTS-235 Data should take current timestamp as default

### PR DESCRIPTION
**Pull request by lucasponce**
Minor one.
In our doc it stands "If not assigned, timestamp will be assigned to current time." but it assigned to 0.
/cc @jshaughn

### CODE REVIEW

File: hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java [hunk 1]
```diff
@@ -92,7 +92,7 @@
 
     /** For JSON Construction ONLY */
     public Data() {
-        this(null, 0L, null);
+        this(null, System.currentTimeMillis(), null);
```
Comment (jshaughn) [inline hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java]: What am I missing?  This chains to full constructor where we do set to current timestamp if necessary.  We similarly protect in setTimestamp().  Why is this change necessary?

Comment (lucasponce) [inline hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java]: Because in the REST endpoint, it is possible to create a Data object without timestamp without any fail, then when no timestamp field is found on the REST it uses this constructor setting the timestamp by default to 0 instead to current time.

Comment (lucasponce) [inline hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/data/Data.java]: You are right, the constructor and setTimestamp() are properly protected.
This patch is useless.
I will close it and investigate further what could be the root cause.
